src/target/riscv/riscv-013.h

```diff
@@ -0,0 +1,22 @@
+/* SPDX-License-Identifier: GPL-2.0-or-later */
+
+#ifndef OPENOCD_TARGET_RISCV_RISCV_013_H
+#define OPENOCD_TARGET_RISCV_RISCV_013_H
+
+#include "riscv.h"
+
+/**
+ * TODO: This functions will be and replaced here by specific access methods
+ * (e.g. abstract commands, program buffer writes and so on.), while specifics
+ * on how to use them to get a register's value will be in `riscv-013_reg.c`.
```

Comment:
Detail: It seems there are some words missing in the first sentence, and as a result of that, I am not sure I have understood the comment.

Reply:
Addressed.

Reply:
Thanks.